### ISSUE TITLE
After not using Leapp for a while, get stuck with a blurred screen

### ISSUE THREAD

**Issue by suspiration** [Noovolari/leapp#111]
**Describe the bug**
After not using Leapp for a while and leaving it in background.  When opening the app again, it'll show a blurred screen without a way to get out.  It seems to be trying to show the MFA modal but keeps getting into a loop where it closes itself.

**Leapp Version**
Add the version of your Leapp (you can find it under taskbar -> about or Leapp -> about on MacOS X)

**To Reproduce**
1. Open Leapp and select an account to generate the AWS auth
2. Do not close Leapp and wait
3. Usually happens after 2-3hours.  Or even after computer goes to sleep.
4. Open Leapp again and you'll see a blurred screen.

**Expected behavior**
Leapp should show the main screen without being blurred.

**Screenshots**
![image](https://user-images.githubusercontent.com/4315914/115429223-11d29580-a1b8-11eb-8896-7b432e93bbea.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - OS Version 11.2.3
 - Leapp Version 0.4.7

**Additional context**
Logs shows a bunch of the below.

```
[2021-04-20 09:06:09.548] [info] [renderer] execute from Leapp:  {
  error: Error: Command failed: /usr/local/bin/az account clear 2>&1

      at ChildProcess.exithandler (child_process.js:312:12)
      at ChildProcess.emit (events.js:310:20)
      at maybeClose (internal/child_process.js:1021:16)
      at Socket.<anonymous> (internal/child_process.js:443:11)
      at Socket.emit (events.js:310:20)
      at Pipe.<anonymous> (net.js:672:12) {
    killed: false,
    code: 127,
    signal: null,
    cmd: '/usr/local/bin/az account clear 2>&1'
  },
  standardout: '/bin/sh: /usr/local/bin/az: No such file or directory\n',
  standarderror: ''
}
[2021-04-20 09:06:10.553] [info] [renderer] execute from Leapp:  {
  error: Error: Command failed: /usr/local/bin/az account clear 2>&1

      at ChildProcess.exithandler (child_process.js:312:12)
      at ChildProcess.emit (events.js:310:20)
      at maybeClose (internal/child_process.js:1021:16)
      at Socket.<anonymous> (internal/child_process.js:443:11)
      at Socket.emit (events.js:310:20)
      at Pipe.<anonymous> (net.js:672:12) {
    killed: false,
    code: 127,
    signal: null,
    cmd: '/usr/local/bin/az account clear 2>&1'
  },
  standardout: '/bin/sh: /usr/local/bin/az: No such file or directory\n',
  standarderror: ''
}
```

**Comment by andreacavagna01** [comment]
Thanks for reporting, clearly an unhandled error, we will look at it after the next release

**Comment by andreacavagna01** [comment]
fixed in v0.6.0
